feat: adds organisations docs (#261)
bcnmy/docs: docs/dashboard/organisations.md

---
sidebar_label: "Organisations"
sidebar_position: 5
---

> # Organisation Overview
> 
> Organisation feature is designed to enhance collaboration and streamline team management within your workspace. This feature allows you to invite new team members to your organisation and manage their access permissions efficiently. Whether you're looking to onboard new hires, collaborate with external partners, or reorganize teams, our tools are crafted to provide you with a seamless management experience.

Reviewer: ```suggestion
# Organization Overview

The organization feature is designed to enhance collaboration and streamline team management within your workspace. 
This feature allows you to invite new team members to your organization and manage their access permissions. Whether you're looking to onboard new hires, collaborate with external partners, or reorganize teams, this feature was designed to provide you with a seamless management experience.
```

> ## Inviting Team Members
> 
> Inviting colleagues to join your organisation is straightforward. Follow these simple steps to send an invitation:
> 
> 1. Click on user settings dropdown and select 'Account Settings'.
> 2. Navigate to 'Team' page.
> 3. Click on 'Invite user' button.
> 4. Enter the email address of the individuals you wish to invite.
> 5. Select the appropriate role or permissions from the predefined list to ensure each team member receives the access they require.
> 6. Send the invitation by clicking the 'Invite' button.
> 
> Once invited, team members will receive an email with a link to join the organization. They will need to follow the link to set up their account and gain access to the resources permitted by their assigned role.

Reviewer: ```suggestion
## Inviting Team Members

Inviting colleagues to join your organization is straightforward. Follow these simple steps to send an invitation:

1. Click on the user settings dropdown and select 'Account Settings'.
2. Navigate to 'Team' page.
3. Click on '+ Invite User' button.
4. Enter the email address of the individuals you wish to invite.
5. Select the appropriate role from the predefined list to ensure each team member receives the access they require.
6. Send the invitation by clicking the 'Invite' button.

You can invite more team members at once by clicking '+ Add More' and following steps 4 to 6.

Once invited, team members will receive an email with a link to join the organization. 
They will need to follow the link to set up their account and gain access to the resources permitted by their assigned role.
```

'+ Invite User' on the UI should also be capitalized for consistency with the rest of the UI.
'+ Add More' should not be FULL CAPS.

> ## Role Permissions
> 
> Roles are defined as follows:
> 
> - `Admin`: Admins have full access to all settings and features, including the ability to invite new members, change any member’s role, manage permissions, and view and edit all projects and financial settings.
> 
> - `Editor`: Editors can access and modify all projects and settings except for team member management. This role is suited for team members who need to make substantial contributions to projects but do not need control over user roles and permissions.
> 
> - `Viewer`: Viewers have read-only access to all projects, meaning they cannot alter any project content. However, Viewers have the ability to deposit funds into the paymaster account, supporting financial operations without modifying project data.

Reviewer: ```suggestion
## Role Permissions

Roles are defined as follows:

- `Admin`: Admins have full access to all settings and features, including the ability to invite new members, change any member’s role, manage permissions, and view and edit all projects and financial settings.

- `Editor`: Editors can access and modify all projects and settings except for team member management. This role is suited for team members who need to make substantial contributions to projects but do not need control over user roles and permissions.

- `Viewer`: Viewers have read-only access to all projects, meaning they cannot alter any project content. However, Viewers can deposit funds into the paymaster account, supporting financial operations without modifying project data.
```
